src/encoded/static/components/motifs.js

```diff
@@ -145,20 +149,34 @@ export class MotifElement extends React.Component {
                     : null}
                     {!(this.props.shortened) ?
                         <React.Fragment>
-                            {(Object.keys(footprintList).length > 0) ?
+                            {(footprintsLength > 0) ?
                                 <p>
                                     <span className="motif-label">{footprintsLabel}</span>
-                                    {Object.keys(footprintList).map((d, dIndex) => <a key={d} href={footprintList[d]}>{d}{dIndex === (Object.keys(footprintList).length - 1) ? '' : ', '}</a>)}
+                                    {footprintKeysSorted.map((d, dIndex) => <a key={d} href={footprintList[d]}>{d}{dIndex === (footprintsLength - 1) ? '' : ', '}</a>)}
                                 </p>
                             : null}
-                            {(Object.keys(pwmList).length > 0) ?
+                            {(pwmsLength > 0) ?
                                 <p>
                                     <span className="motif-label">{pwmsLabel}</span>
                                     {Object.keys(pwmList).map((d, dIndex) => <a key={d} href={pwmList[d]}>{d}{dIndex === (Object.keys(pwmList).length - 1) ? '' : ', '}</a>)}
                                 </p>
                             : null}
                         </React.Fragment>
-                    : null}
+                    :
+                        <React.Fragment>
```

Comment:
Valid but the old way. The new way is to use "<>". See- https://reactjs.org/docs/fragments.html